### PR TITLE
Fix links

### PR DESCRIPTION
**Pull request by rebekaburnett**

### CODE REVIEW

**Comment by rebekaburnett** [comment]
@tht13 This PR doesn't change the content or organization.  It just gives link targets and metadata so that I can import it into docusaurus.

File: docs/api/Customer-Present-Payments.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
+<!-- START_METADATA
+---
+title: Customer Preset Payments
+sidebar_position: 150
+---
+END_METADATA -->
 
+# Customer-Present Payments
 
+When using the ePayments API to process payments in a physical setting, from a POS device, or in a restaurant for example, the payment must be flagged with `$.customerInteraction=CUSTOMER_PRESENT`.
```
Comment (cloveras) [inline docs/api/Customer-Present-Payments.md:10]: The syntax seems language-specific, but this is a REST API, so I suggest:
```suggestion
When using the ePayments API to process payments in a physical setting, from a POS device, or in a restaurant for example, the payment must be flagged with `customerInteraction=CUSTOMER_PRESENT`.
```

File: docs/api/Getting-Started.md [hunk 2]
```diff
@@ -279,7 +281,11 @@ curl https://apitest.vipps.no/epayment/v1/payments/UNIQUE-PAYMENT-REFERENCE/capt
 }'
 ```
 
-Adjustments to a payment (capture, refund etc) as async. You will get a `202 Accepted` response with no body if the action is valid. A callback will be sent once the capture is completed. Additionally polling on [Get Payment]() can be done. Once capture is completed the `Payment` object will be updated to reflect this.
+Adjustments to a payment (capture, refund etc) as async.
+You will get a `202 Accepted` response with no body if the action is valid.
```
Comment (cloveras) [inline docs/api/Getting-Started.md:285]:
```suggestion
You will get a `HTTP 202 Accepted` response with no body if the action is valid.
```

File: docs/api/Logistics.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
+<!-- START_METADATA
+---
+draft: True
```
Comment (cloveras) [inline docs/api/Logistics.md:3]: Lowercase t, right?
```suggestion
draft: true
```

**Review by OFollan — APPROVED** [review]
Screenshot 2022-09-23 at 13 49 51

File: docs/api/Logistics.md (hunk 3 above)
Comment (rebekaburnett) [reply docs/api/Logistics.md]: Yes. I fill fix in all the files

**Pull request merged**